api/controller/contestants.go

```diff
@@ -36,23 +36,25 @@ func GetSingleContestant(w http.ResponseWriter, r *http.Request) {
 		return
```

Comment:
I see that you're using functions to return specific errors, what if you refactor it a little bit to have only one function that would return the error, that would receive as a param the status code you want to return? Like `returnError(resp, w, err, http.StatusBadRequest)`